#! /bin/sh -e
## 21_author_locale.dpatch by Andreas Mueller <amu@canonical.com>
##
## DP: Remove cmdline from the isofs header

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
@DPATCH@
diff -urNad cdrtools-2.01.01~/cdrecord/cdrecord.c cdrtools-2.01.01/cdrecord/cdrecord.c
--- cdrtools-2.01.01~/cdrecord/cdrecord.c	2006-03-19 21:17:52.731310256 +0100
+++ cdrtools-2.01.01/cdrecord/cdrecord.c	2006-03-19 21:18:24.013554632 +0100
@@ -366,7 +366,7 @@
 #	define	CLONE_TITLE	""
 #endif
 	if ((flags & F_MSINFO) == 0 || lverbose || flags & F_VERSION) {
-		printf("Cdrecord%s%s %s (%s-%s-%s) Copyright (C) 1995-2006 Jörg Schilling\n",
+		printf("Cdrecord%s%s %s (%s-%s-%s) Copyright (C) 1995-2006 Joerg Schilling\n",
 								PRODVD_TITLE,
 								CLONE_TITLE,
 								cdr_version,
